.github/workflows/sandbox-images.yaml

```
@@ -490,49 +460,85 @@ jobs:
+          ((${#containers[@]} == 0)) || docker rm -f "${containers[@]}" >/dev/null || cleanup_failed=1
+          mapfile -t volumes < <(docker volume ls -q --filter "label=$label")
+          ((${#volumes[@]} == 0)) || docker volume rm -f "${volumes[@]}" >/dev/null || cleanup_failed=1
+          ! docker ps -aq --filter "label=$label" | grep -q . || cleanup_failed=1
+          ! docker volume ls -q --filter "label=$label" | grep -q . || cleanup_failed=1
+          ((cleanup_failed == 0)) || exit 1
+
+      - name: Upload OpenClaw managed-image security evidence
+        if: ${{ always() }}
         uses: ./.github/actions/upload-e2e-artifacts
+        with:
+          artifact-name: managed-image-openclaw-security-evidence
```

actionlint (1.7.12)

[error] input "artifact-name" is not defined in action "upload-e2e-artifacts" defined at "./.github/actions/upload-e2e-artifacts". available inputs are "name", "path"

(action)